### PR TITLE
add easyblock for binutils (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
see also discussion in https://github.com/hpcugent/easybuild-easyconfigs/pull/2524

cc @wpoely86, @ocaisa

### CODE REVIEW

File: easybuild/easyblocks/b/binutils.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+from easybuild.tools.systemtools import get_shared_lib_ext
+
+
+class EB_binutils(ConfigureMake):
+    """Support for building/installing binutils."""
+
+    def configure_step(self):
+        """Custom configuration procedure for binutils: statically link to zlib, configure options."""
+
+        libdirs = ['/lib', '/lib64', '/usr/lib', '/usr/lib64', '/usr/lib/x86_64-linux-gnu/']
```
Comment (wpoely86) [inline easybuild/easyblocks/b/binutils.py]: check if `/lib` and `/usr/lib` are the same and only add one if needed (same for `lib64`)

**Comment by wpoely86** [comment]
:+1:

**Comment by rjeschmi** [comment]
It looks fine.

probably a good use of RPATH somewhere in here.

The sanity check is probably ok for now, but it would be better to have the sanity check I need for RPATH so we can check in a more robust way.

this is all non-blocking though, just ideas for the future.

**Comment by boegel** [comment]
should be good to go now, see test reports in https://github.com/hpcugent/easybuild-easyconfigs/pull/3006#issuecomment-217641650 (existing binutils easyconfigs) and https://github.com/hpcugent/easybuild-easyconfigs/pull/2524#issuecomment-217652335 (binutils 2.26)

**Comment by wpoely86** [comment]
:+1:

**Comment by boegel** [comment]
Thanks for the review @wpoely86 and @rjeschmi!

**Pull request merged**